README.md

```diff
@@ -55,11 +56,11 @@ Lowercase letters are typically used to begin variable names.
 - Additionally, unit testing encourages a more flexible and reusable code architecture. Developers are compelled to consider the different parts of their code in isolation and how they interact with one another when they design unit tests. As a result, the codebase is better planned and easier to maintain.[https://www.techtarget.com/searchsoftwarequality/answer/Is-unit-testing-an-important-aspect-of-software-development]
 
 # Community code
-- Find smallest nuber in matrix [https://github.com/codinasion/program/issues/5129].
+- [Write a Java program to find smallest element in a matrix #5129](https://github.com/codinasion/program/issues/5129).
```

Comment:
As with the above comment: title first in the square brackets, link second in the rounded parentheses.